src/components/ModalFlows/SupportProposal/SupportProposalScreens.js

```diff
@@ -0,0 +1,52 @@
+import React, { useMemo, useState, useCallback } from 'react'
+import PropTypes from 'prop-types'
+import ModalFlowBase from '../ModalFlowBase'
+import SupportProposal from './SupportProposal'
+import useActions from '../../../hooks/useActions'
+
+function SupportProposalScreens({ versionId, id, onDone, stakeProposal }) {
+  const actions = useActions()
+  const [transactions, setTransactions] = useState([])
+
+  const getTransactions = useCallback(
+    async onComplete => {
+      await actions.agreementActions.signAgreement({ versionId }, intent => {
```

Comment:
Step: 2 after changing the action you also need to change here to call the new action